Dockerfile

```diff
@@ -58,3 +58,10 @@ RUN SWIFT_URL=https://swift.org/builds/$SWIFT_BRANCH/$(echo "$SWIFT_PLATFORM" |
 
 # Print Installed Swift Version
 RUN swift --version
+
+WORKDIR /code
+ONBUILD COPY Package.swift /code/
+ONBUILD RUN swift build -c release
```

Comment:
If you could make the build configuration an `ARG` that defaults to `release` then it means people can parametrize the docker build to debug as necessary.